amida-tech/indaba-client
# Feature/INBA-406 Messaging Connect

#### What's this PR do?
Connects message delete and archive to the backend.

#### Related JIRA tickets:
[INBA-406](https://jira.amida-tech.com/browse/INBA-406)

See [INBA-456](https://jira.amida-tech.com/browse/INBA-456) for last remaining backend connections to make. They require additional endpoints on the backend ([SER-105](https://jira.amida-tech.com/browse/SER-105), [SER-107](https://jira.amida-tech.com/browse/SER-107))

#### How should this be manually tested?
1. Generate some messages by sending them
1. Archive one
1. Check that the archive flag is set in the messaging service database
1. Check that the message goes to the archived list immediately and on reload
1. Delete on (this will fail, see [SER-106](https://jira.amida-tech.com/browse/SER-106). You can check that delete actually works by fixing the scoping issue in the messaging service temporarily. Delete from the client will work when that ticket is closed

#### Any background context you want to provide?
#### Screenshots (if appropriate):
